package.properties

```
@@ -4,7 +4,13 @@ email.smtp.url=smtp.uta.fi
 email.address.weto=weto@sis.uta.fi
 
 # Notification settings
-notification.emailInterval.minutes=60
+
+notification.emailInterval.minutes=5
```

For testing purposes in vagrant it's okay to lower the emailInterval, but in the reference properties file it should be at least an hour
```suggestion
notification.emailInterval.minutes=60
```